index.js

```diff
@@ -166,7 +166,10 @@ FastForEach.prototype.onArrayChange = function (changeSet) {
   // Once a change is registered, the ticking count-down starts for the processQueue.
   if (this.changeQueue.length > 0 && !this.rendering_queued) {
     this.rendering_queued = true;
-    FastForEach.animateFrame.call(window, function () { self.processQueue(); });
+    if (isInitial)
```

Comment:
For consistency of style, the conditional bits should probably have the `{` and `}` :)